tantalus/migrations/0082_auto_20181130_2330.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-11-30 23:30
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def data_migrate(apps, schema_editor):
+    seqds = apps.get_model('tantalus','SequenceDataset')
+    vocab = apps.get_model('tantalus','ReferenceGenome')
+
+    recs = seqds.objects.values_list('temp',flat=True).distinct()
+
+    print("\n\n%s;\n  N=%i;\n" % (str(recs.query),recs.count()))
+
+    #fill vocabulary, ignore existing values
+    for x in recs:
+        print(x)
+        vocab.objects.get_or_create(reference_genome=x)
+
+    recs = seqds.objects.all()
+
+    #save values in the temporary field
+    print("\n\n%s;\n  N=%i;\n" % (str(recs.query),recs.count()))
+    for r in recs:
+        v = vocab.objects.get(reference_genome=r.temp)
+        r.reference_genome=v
+        if (0 == (r.id % 113)):
+            print("----- %s; %s;" % (r.id,r.reference_genome.id))
+        r.save()
+
+
+def do0(apps, schema_editor):
+    pass
```

Comment:
what is this?

Reply:
Empty function, required to rollback migration, without it rollback fails.